api_fhir_r4/views/fhir/practitioner.py

```diff
@@ -71,11 +71,13 @@ def get_queryset(self):
         return ClaimAdmin.objects
 
     def _ca_queryset(self):
-        queryset = ClaimAdmin.objects.filter(validity_to__isnull=True).order_by('validity_from')
+        queryset = ClaimAdmin\
+            .objects\
+            .filter(validity_to__isnull=True).all()
         return ValidityFromRequestParameterFilter(self.request).filter_queryset(queryset)
 
     def _eo_queryset(self):
-        queryset = Officer.objects.filter(validity_to__isnull=True).order_by('validity_from')
+        queryset = Officer.objects.filter(validity_to__isnull=True).order_by('validity_from').all()
```

Comment:
A small stylistic note as to why here above is formatted and here it is not? I think is good to keep one chosen approach with styling.